bears/general/GenericHelpers.py

```diff
@@ -0,0 +1,162 @@
+from coala_utils.string_processing.Core import unescaped_search_for
```

Comment:
also what name do you suggest? it's got to do with finding ranges of things specified within a start and end.